test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -84,6 +84,11 @@ Describe "Basic FileSystem Provider Tests" -Tags "CI" {
             $dirContents.Count | Should Be 2
         }
 
+        It "Verity Get-ChildItem can get the name of a specified item." {
+            $fileName = Get-ChildItem $testFile -Name
+            $fileName | Should BeExactly $fileName
```

Comment:
Yikes! Bad typo on my part. That should have been `$filename | Should BeExactly $testFile`. Is making that change sufficient, or would you prefer the dual-call method?